### DISCUSSION TITLE
[symfont/notifier] The "brevo" scheme is not supported (with reproducer repo)

### DISCUSSION THREAD

**Discussion by Snowbaha** [symfony/symfony#57339] [Q&A] [answered]
### Symfony version(s) affected

7.1.1

### Description

I updated a project from Symfony 6.4 to 7.1 and my SMS notification didn't work. 
I have this error : 
![image](https://github.com/symfony/symfony/assets/3284722/63cdbcc8-1ee4-4f89-be74-a76cd0c83a9b)

I searched deeper in the vendor and it looks the factories didn't find the brevo...

If I come back to Symfony 6.4 without change, it is working.

I am becoming crazy... I didn't read BC in Notifier in Symfony 7.1.

Can you help me?

### How to reproduce

You can clone this repo and run it: https://github.com/Snowbaha/BrevoNotifierErrorSchema

**Comment by xabbuh** [comment, marked as answer]
Can you please check if #57348 fixes this for you?

**Comment by Snowbaha** [comment]
Oh yes I changed the file in my reproducer and I have this error (normal because the key is not setup) :
![image](https://github.com/symfony/symfony/assets/3284722/17a4cc97-f6bc-40b8-bffb-2851293e633a)


Thank you for your help!
